templates/terraform/examples/artifact_registry_repository_cmek.tf.erb

```diff
@@ -0,0 +1,9 @@
+resource "google_artifact_registry_repository" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+
+  location = "us-central1"
+  repository_id = "<%= ctx[:vars]['repository_id'] %>"
+  description = "<%= ctx[:description] %>"
```

Comment:
Instead of `<%= ctx[:description] %>`, you'll just want to put the hard-coded value in here. This  actually ended up generating an empty description, since the value of the description was set in `ctx[:vars]` and not in `ctx` outright: https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-3658-old..auto-pr-3658#diff-9fbc35188620ea4ba754beea15012972R85

Reply:
gotcha, thanks!